### PR TITLE
47377_supervisor_assign_supervisor_to_course

### PR DESCRIPTION
**Pull request by yasin-buet**
screenshot from 2015-12-09 12 00 16
screenshot from 2015-12-09 12 00 26

### CODE REVIEW

**Comment by asifbuetcse** [comment]
ok

File: app/Http/Controllers/Supervisor/CoursesController.php [hunk 1]
```diff
@@ -75,7 +77,15 @@ public function edit($id)
...
     {
-        //
+        $course = Course::findOrFail($id);
+        $this->validate($request, [
+            'name' => 'required',
+            'description' => 'required',
+            'supervisor_id' => 'required'
+        ]);
+        $input = $request->all();
+        $course->fill($input)->save();
```
Comment (dinhhoanglong91) [inline app/Http/Controllers/Supervisor/CoursesController.php]:
```php
$course->save($input);
```

File: resources/views/supervisor/courses/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@extends('layouts.supervisor')
+@section('content')
+<h1>Edit Course - {{ $course->name }}</h1>
+<p class="lead">Edit this course below. <a href="{{ route('supervisor.courses.index') }}">Go back to all courses.</a></p>
```
Comment (dinhhoanglong91) [inline resources/views/supervisor/courses/edit.blade.php]: Break line

File: resources/views/supervisor/courses/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{!! Form::model($course, [
+    'method' => 'PATCH',
+    'route' => ['supervisor.courses.update', $course->id]
+]) !!}
+<div class="form-group">
```
Comment (dinhhoanglong91) [inline resources/views/supervisor/courses/edit.blade.php]: Indent here

File: resources/views/supervisor/courses/edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+</div>
+<div class="form-group">
+    {!! Form::label('description', 'Description:', ['class' => 'control-label']) !!}
+    {!! Form::textarea('description', null, ['class' => 'form-control']) !!}
+</div>
+<div>
+    {!! Form::label('supervisor_id', 'Select A Supervisor:', ['class' => 'control-label']) !!}
+    {!! Form::select('supervisor_id', $supervisors) !!}
+</div>
+{!! Form::submit('Update Task', ['class' => 'btn btn-primary']) !!}
```
Comment (dinhhoanglong91) [inline resources/views/supervisor/courses/edit.blade.php]: `Task` or `Course`?

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**